features/admin/adding_gift_card.feature

```diff
@@ -0,0 +1,34 @@
+@managing_gift_cards
+Feature: Creating a gift card
+  In order to have gift cards in the shop
+  As an Administrator
+  I want to create a new gift card
+
+  Background:
+    Given the store operates on a single channel in "United States"
+    And I am logged in as an administrator
+    And there is a customer account "john-doe@setono.com"
+
+  @api
+  Scenario: Creating a gift card
+    When I want to create a new gift card
+    And I specify its code as "GIFT-CARD-100"
+    And I do not specify its customer
+    And I specify its amount as "10000"
+    And I specify its currency code as "USD"
```

Comment:
Is this the way to do it? A more non-technical would write `And I specify its amount as "$100"`

Reply:
Ok, done but the currency code line is still needed (also done in Sylius core)